public/components/marketing-forms/5.html

```diff
@@ -0,0 +1,104 @@
+            <div>
+              <label for="username" class="sr-only">Username</label>
+              <input
+                class="w-full rounded border-gray-200 p-3 text-sm"
+                placeholder="Username"
+                type="text"
+                id="username"
+              />
+            </div>
+
+            <div>
+              <label for="email" class="sr-only">Email</label>
+              <input
+                class="w-full rounded border-gray-200 p-3 text-sm"
+                placeholder="Email"
+                type="email"
+                id="email"
+              />
+            </div>
+            <div>
+              <label for="bio" class="sr-only">Bio</label>
+              <textarea
+                id="bio"
+                cols="30"
+                class="w-full rounded border-gray-200 p-3 text-sm"
+                placeholder="Bio"
+              ></textarea>
+            </div>
+          </div>
+        </div>
+        <div class="grid gap-4 pt-8 md:grid-cols-6">
+          <hgroup class="md:col-span-2">
+            <h3 class="text-lg font-medium">Notifications</h3>
+            <p class="text-sm text-gray-500">
+              Configure how you receive notifications
+            </p>
+          </hgroup>
+          <div class="space-y-3 md:col-span-4 md:max-w-lg">
+            <fieldset>
+              <legend class="text-sm font-medium text-gray-500">
+                Notify me about...
+              </legend>
+              <div class="mt-3 space-y-3">
+                <div class="flex items-center gap-2">
+                  <input id="all" name="notification-type" type="radio" />
+                  <label for="all" class="text-sm text-gray-500"
+                    >All new messages</label
+                  >
+                </div>
+                <div class="flex items-center gap-2">
+                  <input id="mentions" name="notification-type" type="radio" />
+                  <label for="mentions" class="text-sm text-gray-500"
+                    >Direct messages and mentions</label
+                  >
+                </div>
+                <div class="flex items-center gap-2">
+                  <input id="none" name="notification-type" type="radio" />
+                  <label for="none" class="text-sm text-gray-500"
+                    >Nothing</label
+                  >
+                </div>
+              </div>
+            </fieldset>
+          </div>
+        </div>
+      </div>
+      <div class="flex items-center justify-end gap-3">
+        <button
+          type="button"
+          class="inline-block rounded-lg bg-gray-300 px-6 py-3 font-medium text-gray-800 hover:opacity-80"
+        >
+          Cancel
+        </button>
+        <button
+          type="submit"
+          class="inline-block rounded-lg bg-gray-800 px-6 py-3 font-medium text-gray-100 hover:opacity-80"
+        >
+          Save
+        </button>
```

Comment:
These are huge! 😅 Can you use buttons from the HyperUI library, please?

Reply:
Ok no problemo 😁

Reply:
They are more inline with what you'd expect for a button IMO, the font size on yours are making them look a little chunky. Which is fine but I'd prefer a slimmer looking button, like the ones currently on HyperUI.